DEVELOPMENT.md

```diff
@@ -0,0 +1,78 @@
+# Developement
+
+## Feature Release Flow
+
+### Branching Strategy
+
+- The `master` branch is intended to only contain features for the immediately upcoming release, whether it is a Height Coordinated Upgrade or a Spork
+- During development, all features should live on a feature branch.
+- For small features, this will be simple working branch, i.e. `kan/1-fix-known-issue`
+- For larger features, this may be a shared feature branch with other team members.
+
+### Upgrade Path Eligibility
+
+- When a feature branch is ready to be merged, it must be determined if the desired upgrade path onto Mainnet. The options are:
+    - Height Coordinated Upgrade (HCU)
+        - No protocol level breaking changes
+        - No state migrations
+        - Execution state/path changes allowed
+        - Resource optimizations are okay
+        - Cadence upgrades which could cause execution state fork (likely any Cadence upgrade except for trivial changes)
+    - Spork
+        - Protocol level breaking change
+        - State migrations required
+- All HCU upgrades can go directly into the `master` branch
+- All spork upgrades must live on a their own feature branch until the last HCU before the spork as been performed (usually approximately 1 month before the Spork.
+    - It is the responsibility of the DRI to keep this feature branch in a mergeable state
+    - Suggestion: once a sprint, merge `master` into the feature branch
+
+### End of Release Cycle
+
+- At the end of every release cycle, we will tag a commit that includes all desired features to be Released
+- This commit will be tagged according to semantic versioning guidelines
+- Release notes must be written up, describing all features included in this tag
+
+### Benchmark Testing
+
+[Benchmarking](https://www.notion.so/Benchmarking-e3d89e3aadb44b0787da9bb7703b0dae?pvs=21)
+
+- All features on the release candidate tag must undergo testing on `benchmarknet`
+
+### Testnet
+
+- The current schedule is the Wednesdays two weeks before the corresponding mainnet spork
+- Features should aim to live on Testnet for at least a week before making it to Mainnet
+
+### Mainnet
+
+- Features must live on Testnet for two week before making it to Mainnet
```

Comment:
This conflicts with the previous statement:
> - Features should aim to live on Testnet for at least a week before making it to Mainnet

should this be updated to one week, or does it need more qualification?